api/lib/middleware/req-logger.js

```diff
@@ -8,6 +8,16 @@ function reqLog( req, res, next ) {
     log.debug( 'request body: ' + JSON.stringify( req.body ) );
     log.trace( req );
 
+    const baseSend = res.send;
+
+    res.send = ( data ) => {
+      log.debug( 'response payload: ' + JSON.stringify( data ) );
+
+      res.send = baseSend;
```

Comment:
res.send = res.send ?

Reply:
Lol yeah this is kinda fucked. We save it as `baseSend` so that we can use it later after we have redefined it to have the logger. We can't call `baseSend` because it wont have access to the other things in the `res` object that it needs to use.